pages/linux/x11vnc.md

```diff
@@ -0,0 +1,23 @@
+# x11vnc
+
+> A VNC server that will enable VNC on an existing display.
+
+- Launch a VNC server that allows multiple clients to connect:
+
+`x11vnc -shared`
+
+- Launch a VNC server that will not exist after a client disconnects:
+
+`x11vnc -forever -viewonly`
```

Comment:
What they do is:  
* In **normal** mode - you start the `x11vnc` server and 1 client gets to connect.  When it disconnects the software shuts down
* In **viewonly** mode you cannot interact with the server, you can only look at what is happening
* In **shared** mode multiple clients can connect and either view (view only) or interact (normal mode)
* In **forever** mode - once the last client disconnects the software does not terminate.

Any suggestions on how to communicate this information across?